content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -28,28 +28,22 @@ DNS is a built-in Kubernetes service launched automatically
 using the addon manager
 [cluster add-on](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/README.md).
 
-The running DNS Pod holds 3 containers:
+Following Kubernetes v1.12, CoreDNS is declared the default DNS Server instead of kube-dns. In kubeadm, this change was made
```

Comment:
"Following Kubernetes v1.12, CoreDNS is declared the default DNS Server instead of kube-dns. In kubeadm, this change was made in Kubernetes 1.11, but other installers have waited until 1.12. You should refer to the documentation of other installers to verify which is the default DNS server."

Perhaps change this to ...

"As of Kubernetes v1.12, CoreDNS is the recommended DNS Server, replacing kube-dns.  However, kube-dns may still be installed by default with certain Kubernetes installer tools. You can refer to the documentation your installer to know which DNS server is installed by default."